src/app.css

```diff
@@ -1,8 +1,48 @@
-:root {
-  font-family: Arial, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu,
-    Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-}
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
question (non-blocking): why do you import the font in CSS? I believe it is less performant since the CSS has to be loaded and parsed before the request for the font files is made.

Reply:
Didn't do it consciously, just copied to code from GFonts. I will move it to `<link>`.